recipes/configs/phi3/mini_full.yaml

```diff
@@ -20,12 +20,12 @@
 # Tokenizer
 tokenizer:
   _component_: torchtune.models.phi3.phi3_tokenizer
-  path: /tmp/cpts/Phi-3-mini-4k-instruct/tokenizer.model
+  path: /tmp/Phi-3-mini-4k-instruct/tokenizer.model
 
 # Dataset
 dataset:
-  _component_: torchtune.datasets.alpaca_dataset
-  train_on_input: True
+  _component_: torchtune.datasets.slimorca_dataset
```

Comment:
I changed the default dataset here to slimorca b/c the main use case of this model is chat style

Reply:
Awesome catch!